spec/install/gemfile/ruby_spec.rb

```diff
@@ -22,4 +26,64 @@
     expect(exitstatus).to eq(0) if exitstatus
     should_be_installed "rack-obama 1.0"
   end
+
+  it "allows removing the ruby version requirement" do
+    install_gemfile <<-G
+      source "file://#{gem_repo1}"
+      ruby "~> #{RUBY_VERSION}"
+      gem "rack"
+    G
+
+    expect(lockfile).to include("RUBY VERSION")
+
+    install_gemfile <<-G
+      source "file://#{gem_repo1}"
+      gem "rack"
+    G
+
+    should_be_installed "rack 1.0.0"
+    expect(lockfile).not_to include("RUBY VERSION")
+  end
+
+  it "allows changing the ruby version requirement to something compatible" do
+    install_gemfile <<-G
+      source "file://#{gem_repo1}"
+      ruby ">= 1.0.0"
+      gem "rack"
+    G
+
+    expect(locked_ruby_version).to eq(Bundler::RubyVersion.system)
+
+    simulate_ruby_version "5100"
```

Comment:
These invocations don't restore the RUBY_VERSION and ENV["BUNDLER_SPEC_RUBY_VERSION"] afterwards right? Should we be invoking the following calls/assertions within blocks passed to `simulate_ruby_version`?